composer.json

```diff
@@ -14,7 +14,6 @@
 		"hoa/compiler": "3.17.08.08",
 		"hoa/exception": "^1.0",
 		"hoa/file": "1.17.07.11",
-		"hoa/regex": "1.17.01.13",
```

Comment:
`hoa/regex` is used and I want to keep it here (I know your tool cannot possibly detect it): https://github.com/phpstan/phpstan-src/blob/fd0ded1b5422acb941201fd3dd42e72d94386598/src/DependencyInjection/ValidateIgnoredErrorsExtension.php#L59

Reply:
Thanks, ignored too.